examples/vision/image_classification_from_scratch.py

```diff
@@ -99,19 +100,33 @@
 """
 ## Visualize the data
 
-Here are the first 9 images in the training dataset. As you can see, label 1 is "dog"
-and label 0 is "cat".
+Here are the first 8 images in the training dataset. We visualize this by making
```

Comment:
I suggest to shorten this description:
"Here are the first 8 images in the training dataset, visualized using the KerasCV `plot_image_gallery` utility."